pkg/volume/gce_pd/gce_util.go

```diff
@@ -86,33 +86,30 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (strin
 	// Apply Parameters (case-insensitive). We leave validation of
 	// the values to the cloud provider.
 	diskType := ""
-	zone := ""
+	var PutAdminAndUserRequestsTogetherParams volume.PutAdminAndUserRequestsTogetherParams
 	for k, v := range c.options.Parameters {
 		switch strings.ToLower(k) {
 		case "type":
 			diskType = v
 		case "zone":
-			zone = v
+			PutAdminAndUserRequestsTogetherParams.IsSCZoneSpecified = true
```

Comment:
Does this get cleaner if we pass `c.options` in to `volume.PutAdminAndUserRequestsTogether` ?